apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js

```diff
@@ -96,13 +97,16 @@ function WelcomeTour() {
 }
 
 registerPlugin( 'wpcom-block-editor-nux', {
-	render: () => (
-		<>
-			<WelcomeTour />
-			<PostPublishedModal />
-			<SellerCelebrationModal />
-			<PurchaseNotice />
-			<VideoPressCelebrationModal />
-		</>
-	),
+	render: () => {
+		return (
+			<>
+				<WelcomeTour />
+				<FirstPostPublishedModal />
+				<PostPublishedSharingModal />
+				<SellerCelebrationModal />
```

Comment:
@roo2 FYI, `SellerCelebrationModal` is the model I had mentioned last week as something we should make sure we're not conflicting with. Since these are all grouped together, I'm assuming there's logic here to make sure there aren't multiple shown at the same time, is that right?

Reply:
thanks, I forgot to investigate this actually! 😬 I will investigate today 👀